### PR TITLE
Add html elements, js objects, and submit handler.

### PR DESCRIPTION
**Pull request by amcavinue**

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        fruity: [
+            'Slice of orange',
+            'Dash of cassis',
+            'Cherry on top'
+        ],
+    },
+    preferences = {};
+
+var Bartender = function() {
+    this.createDrink = function() {
```
Comment (rockchalkwushock) [inline scripts.js]: Think you might be trying to use `this.createDrink` as a prototype but that won't work here. You are declaring that `this.createDrink is an anonymous function`. You won't be able to apply `this.createDrink` to your `Bartender` object. 
You would want to day:

```
var Bartender = function(x, y, z) {
 this.thing1 = x,
 this.thing2 = y,
 this.thing3 = z
}

Bartender.prototype.createDrink = function(params) {
 // code here for creating your drink
}

```

You might be doing something different that I've never seen before either!
My repo if you need something to look at:
https://github.com/rockchalkwushock/PirateBartender

**Review by rockchalkwushock — COMMENTED** [review]
Overall looks good so far. Keep things organized and commented well for yourself and those who review your code. Makes for quick understanding of what you were aiming at and quick for error fixes.

File: scripts.js (hunk 1 above)
Comment (amcavinue) [reply scripts.js]: 👍

**Comment by amcavinue** [comment]
Thanks @rockchalkwushock. Your changes have been noted and updated in the code. Cheers!

**Pull request merged**